docs/user_guide/backend_configuration.rst

```diff
@@ -0,0 +1,284 @@
+Backend Configuration
+=====================
+
+NeuroConv offers highly convenient control over the type of file backend and the way its datasets are configured.
```

Comment:
```suggestion
NeuroConv offers convenient control over the type of file backend and the way each dataset is configured.
```

I prefer docs to be a bit more matter-of-fact. "highly" feels a bit sales-y to me here.